CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 ## 8.1.0
```

Comment:
Before creating a new version we list changes under the 'next' line. Only at the stage of a version creation a developer should decide which version pattern to use (minor/major/patch) based on the listed changes, so at that point the list of changes is moved under the version tag